src/loader/struct_util.rs

```diff
@@ -7,14 +7,21 @@
 //
 // SPDX-License-Identifier: Apache-2.0 AND BSD-3-Clause
 
+//! Utility functions for struct manipulation.
```

Comment:
This shouldn't be a public module, we could drop the high level comment (or not, your choice).

Reply:
I think we should wait for #26 to be merged. This wouldn't represent a concern anymore.